cmd/session.go

```diff
@@ -44,8 +44,18 @@ func setupSSHKey(ctx context.Context) (string, []byte, error) {
 		return name, pub, nil
 	}
 
-	// No key details provided, try using ~/.ssh/id_rsa.pub
+	// No key details provided, try using ~/.unweave_global/.ssh/
+	name, pub, err := getFirstPublicKeyInPath(ctx, config.GetUnweaveSSHKeysFolder())
+	if err == nil {
+		return name, pub, nil
+	}
+
+	// No SSH key found, generate a new one
+	return generateSSHKey(ctx)
+}
 
+// Deprecated: we no longer use the idrsa key as default
+func useIdRsaSSHKey(ctx context.Context) (string, []byte, error) {
 	home, err := os.UserHomeDir()
 	if err != nil {
 		return "", nil, err
```

Comment:
Here we remove id_rsa as the first choice of the CLI when fetching the public key to create new containers with. We add the Unweave SSH key, or create it in its place.